epi_scanner/app.py

```diff
@@ -154,14 +167,14 @@ async def update_r0map(q: Q):
     await q.page.save()
     q.page["plot_alt_R0"] = ui.vega_card(
         box="R0_map",
-        title=f"R0 by city in {year}",
+        title=f"",
```

Comment:
Again, why are you removing the titles?

Reply:
Have you looked here: https://drive.google.com/file/d/14bPUpRLdgNgxSyoFzEhGZFWY02k1rygr/view?usp=sharing

For some reason, the h20 wave package doesn't allow changing the plot's title in the vega_card. For this reason, I moved it to viz.py since Altair allows changing it. In order to make the code reproducible, I created an additional parameter called `title`.

I ensured this new page's result was the same as the original but with larger titles (see the drive file). But if you think this will make the code worse, I can forget about these changes and keep everything as was.